index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Technical documentation</title>
+</head>
+    <body>
+        <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+        <nav id="navbar">
+          <header> java</header>
+          <a href="#top" class="nav-link">topic</a>
+          <a href="#numbers" class="nav-link">paragraph</a>
+          <a href="#again" class="nav-link">code</a>
+          <a href="#list" class="nav-link">list</a>    
+        </nav>
+       <main id="main-doc">
+    <header>
+
+      <section class="main-section">
+          <h1>paragraph heading</h1>
+        <p>This a head paragraph </p> 
+      </section>
+      <section class="main-section" id="basement">
+          <h2>basement heading</h2>
+          Basment
+        </section>
+      <section class="main-section" id="walls">
+        <h2>basement heading</h2>
+          <p>Walls</p>
+        </section>
+      <section class="main-section" id="roof">
+        <h2>Roof heading</h2>
+          <p>Roof</p>
+        </section>
+      <section class="main-section" id="heat">
+        <h2>heat heading</h2>
+          <p>Heat</p>
+        </section>  
+    </header>
+
+         <section class="main-section">
+           <h3 id='numbers'>numbers</h3>
+           <p>one</p>
+           <p>Two</p>
+           <p>Three</p>
+           <p>four</p>
+           <p>five</p>
+           <p>six</p>
+           <p>sevev</p>
+           <p>Eight</p>
+           <p>Nine</p>
+           <p>Ten</p>
+         </section>
+    
+         <section class="main-section">
+           <h3 id='again'>again numbers</h3>
+          <code>one</code>
+          <code>two</code>
+          <code>Three</code>
+          <code>four</code>
+          <code>five</code>
+         </section>
+    
+         <section class="main-section">
+           <h3 id='list'>list numbers</h3>
+          <ul>
+            <li>1</li>
+            <li>2</li>
+            <li>3</li>
```

Comment:
should have given each list its own class so that may be in the future I can easily style the project